test/powershell/Modules/Microsoft.PowerShell.Utility/JsonObject.Tests.ps1

```diff
@@ -0,0 +1,53 @@
+﻿# Unit tests for JsonObject
+Describe 'JsonObject' -tags "CI" {
+
+    function ShouldThrow
+    {
+        param (
+            [Parameter(ValueFromPipeline = $true)]
+            $InputObject,
+            [Parameter(Position = 0)]
+            $expectedException
+        )
+
+        try
+        {
+            & $InputObject
+        }
+        catch
+        {
+            $ex = $_.Exception.InnerException
+        }
+
+        $ex | Should Not BeNullOrEmpty
+        $ex.GetType() | Should Be $expectedException
+    }
+
+    It 'empty string' {
+        $errRecord = $null
+        [Microsoft.PowerShell.Commands.JsonObject]::ConvertFromJson("", [ref]$errRecord)
```

Comment:
All test use the same code with different data so it is recommended to use [testcase](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md)
Init code must be moved to `BeforeAll` block. (Use the same link to docs)